packages/api/src/tx/stargate-signing.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  SigningStargateClient,
+  SigningStargateClientOptions,
+} from '@cosmjs/stargate';
+import { OfflineSigner } from '@cosmjs/proto-signing';
+
+/**
+ * createStargateSigningClient returns a SigningStargateClient
+ * to be used for signing and broading transactions.
+ *
+ * @param endpoint - string.
+ * @param signer - OfflineSigner.
+ * @param clientOptions - SigningStargateClientOptions.
+ */
+export async function createStargateSigningClient(
+  endpoint: string,
+  signer: OfflineSigner,
+  clientOptions: SigningStargateClientOptions,
+): Promise<SigningStargateClient> {
+  const defaultClientOptions = {
+    broadcastPollIntervalMs: 300,
```

Comment:
I'm looking forward to playing with this number, to see if we can improve the UX by polling the right amount. This seems like a good duration to start. It's nice we can override it too.